components/style.css

```diff
@@ -229,6 +234,38 @@ i:hover {
     text-align: center;
 }
 
+.logo-dark {
+    display: none;
+}
+
+/* dark-mode */
+
+.dark-mode {
+    background-color: #121212ea;
+    color: var(--background-color);
+    transition: background-color 1s;
+}
+
+body.dark-mode .text, .social-links {
+    color: var(--background-linen);
+}
+
+body.dark-mode .nav-center h4 a {
+    color: #fff9ec;
+}
+
+body.dark-mode .logo {
+    display: none;
+}
+
+body.dark-mode .logo-dark {
+    display: block;
+}
+
+body.dark-mode .hero {
+    
+}
```

Comment:
Empty style block can be removed

Reply:
kept it blank as I am still figuring how to have perfect image for black mode. Cool, I'll remove it for now.